### DISCUSSION TITLE
cURL error 6: getaddrinfo() thread failed to start

### DISCUSSION THREAD

**Discussion by isokaze04** [FreshRSS/FreshRSS#5742] [Q&A] [answered]
**Describe the bug**
get "cURL error 6: getaddrinfo() thread failed to start ...."  error when fetching rss content or adding a new feed.
1.21.0 works fine.

**To Reproduce**
1. Go to '…'
2. Click on '…'
3. Scroll down to '…'

**Expected behavior**

**Screenshots**

**Environment information (please complete the following information):**
- OS: Ubuntu 16.04.3 LTS
- FreshRSS version: 1.22.0
- Installation type: docker (freshrss/freshrss)

**Additional context**

**Comment by Alkarex** [comment, marked as answer]
Could you try with `freshrss/freshrss:alpine`?
It could be due to Ubuntu 16.04 being too old for hosting our Debian 12 image.

**Comment by isokaze04** [comment]
Thanks @Alkarex it works. This issue can be closed now.

**Comment by Alkarex** [comment]
Could you please try one more time with `freshrss/freshrss:edge` just to confirm the problem?

**Reply by isokaze04** [reply]
same error with the edge branch.

**Reply by Alkarex** [reply]
Thanks for the feedback. So Alpine in those situations

**Comment by pettazz** [comment]
I'm using ubuntu 20.04.6 and had the same issue (also works using the :alpine image instead)
